docs/eventing/channels/subscriptions.md

```diff
@@ -23,26 +30,38 @@ kn subscription create <subscription_name> \
 ```
 
 
-- `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default channel that is backed by the Channel resource, you must prefix the channel name with the `<Group:Version:Kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1beta1:KafkaChannel` for a Kafka backed channel.
+- `--channel` specifies the source for cloud events that should be processed.
+You must provide the channel name. If you are not using the default channel that is backed by the
+Channel resource, you must prefix the channel name with the `<Group:Version:Kind>` for the specified
+channel type.
+For example, this is `messaging.knative.dev:v1beta1:KafkaChannel` for a Kafka-backed channel.
 
-- `--sink` specifies the target destination to which the event should be delivered. By default, the `<sink_name>` is interpreted as a Knative service of this name, in the same namespace as the subscription. You can specify the type of the sink by using one of the following prefixes:
+- `--sink` specifies the target destination to which the event should be delivered.
+By default, the `<sink_name>` is interpreted as a Knative service of this name, in the same
+namespace as the subscription.
+You can specify the type of the sink by using one of the following prefixes:
 
     - `ksvc`: A Knative service.
     - `svc`: A Kubernetes Service.
-    - `channel`: A channel that should be used as destination. Only default channel types can be referenced here.
+    - `channel`: A channel that should be used as the destination. You can only reference default
+    channel types here.
     - `broker`: An Eventing broker.
-\
-&nbsp;
-- `--sink-reply` and `--sink-dead-letter` are optional arguments. They can be used to specify where the sink reply will be sent to, and where to send the cloud event in case of a failure, respectively. Both use the same naming conventions for specifying the sink as the `--sink` flag.
+    - `--sink-reply` is an optional argument you can use to specify where the sink reply is sent.
+    It uses the same naming conventions for specifying the sink as the `--sink` flag.
+    - `--sink-dead-letter` is an optional argument you can use to specify where to send the cloud
```

Comment:
Since you're here, we should probably change 'cloud event' to CloudEvent for consistency. And also curious if on L60 'routes events' => 'routes CloudEvents'

Reply:
See above, let's decide on this later for consistency across docs

Reply:
@abrennan89 good point.